.github/workflows/publish-examples.yml

```diff
@@ -56,6 +56,11 @@ jobs:
               continue
             fi
 
+            # web_worker_fib does not compile with trunk. See <https://github.com/thedodd/trunk/issues/40>.
```

Comment:
The referenced issue is closed and supposedly fixed. Is this the correct link?

Reply:
This is the incorrect link. The correct one is https://github.com/thedodd/trunk/issues/46